task_2.py

```diff
@@ -0,0 +1,10 @@
+def task_2(my_dict1,my_dict2):
+    dict3 = my_dict1.copy()
+    dict3.update(my_dict2)
+    for key in my_dict1:
+        if key in my_dict2:
+            dict3[key] = my_dict1[key] + my_dict2[key]
+    print(dict3)
```

Comment:
Leave a line after end of a function. Please follow pep8 guidelines.

Reply:
On it